ses-resources/cephadm

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# Source this to run cephadm-specific validations
+# for the SUSE Enterprise Storage supportutils plugin
+
+
+#############################################################
+rpm_list="$SES_SES_RESOURCES_DIR"/rpm-list
```

Comment:
Should be `rpm_list="$SES_RESOURCES_DIR"/rpm-list` (i.e. remove the extra "SES")